src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,31 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder result = new StringBuilder();
+        result.append("Report for period ").append(dateFrom).append(" - ")
+                .append(dateTo);
+        for (String name : names) {
+            int salaryCount = 0;
+            for (String datum : data) {
+                if (name.equals(datum.substring(11, 11 + name.length()))
```

Comment:
```suggestion
            for (String datum : data) {
            String[] datumParts = datum.split(" ");
                if (name.equals(datum.substring(11, 11 + name.length()))
```
Так буде більш читаймо, ти зможеш звертатись до імені, дати і тд через datumParts[0, 1, ..]
+ не потрібно буде декалька раз за метод сплітити одну строку